sdk/communication/azure-communication-chat/README.md

```diff
@@ -20,7 +20,7 @@ Azure Communication Chat contains the APIs used in chat applications for Azure C
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-communication-chat</artifactId>
-    <version>1.0.0-beta.4</version>
+    <version>1.0.0-beta.7</version>
```

Comment:
```suggestion
    <version>1.0.0-beta.9</version>
```

Reply:
The README should point to the latest release version.

Reply:
After Monday, beta.9 will be the latest